src/types/HashedSBVR.ts

```diff
@@ -0,0 +1,25 @@
+import * as bcrypt from 'bcrypt'
```

Comment:
This doesn't handle the conditional choice between `bcrypt` and `bcryptjs` (bcrypt is preferable but it can be a pain to install, eg windows, or straight up impossible eg in the browser)

The way to do it would be:
```typescript
import * as _bcrypt from 'bcrypt'

let bcrypt: typeof _bcrypt 
try {
 	bcrypt = require('bcrypt')
 } catch (e) {
 	bcrypt = require('bcryptjs')
}
```
\- the way that works is that `import * as _bcrypt from 'bcrypt'` will only actually write out the import to the js file if `_bcrypt` is used, which it never is (avoiding the problem of it not existing at runtime) and instead just import it's type info for compile time.  Having that type info available allows us to still have the full type info available on `bcrypt` whilst using the `let x = require()` style that allows for conditional imports - the `_` prefix specifically is because you have to prefix an `_` for an unused variable in typescript, and it lets us know not to use it 'for real'